diff -up cube-4.2/build-backend/Makefile.in.link cube-4.2/build-backend/Makefile.in
--- cube-4.2/build-backend/Makefile.in.link	2013-08-02 05:20:37.000000000 -0600
+++ cube-4.2/build-backend/Makefile.in	2013-09-27 11:36:20.372970338 -0600
@@ -927,7 +927,7 @@ am__installdirs = "$(DESTDIR)$(libdir)"
 	"$(DESTDIR)$(jardir)" "$(DESTDIR)$(cube_headersdir)" \
 	"$(DESTDIR)$(cubew_headersdir)" "$(DESTDIR)$(includedir)"
 LTLIBRARIES = $(lib_LTLIBRARIES) $(noinst_LTLIBRARIES)
-libcube4_la_LIBADD =
+libcube4_la_LIBADD = -lz
 am__libcube4_la_SOURCES_DIST =  \
 	$(SRC_ROOT)src/cube/src/service/CubeServices.cpp \
 	$(SRC_ROOT)src/cube/src/Cube.cpp \
